src/main/java/ca/corefacility/bioinformatics/irida/ria/web/pipelines/PipelineController.java

```diff
@@ -136,50 +136,6 @@ public PipelineController(SequencingObjectService sequencingObjectService,
 		this.iridaPipelinePluginStyle = iridaPipelinePluginStyle;
```

Comment:
It looks like `iridaPipelinePluginStyle` is no longer used in this class. Could it be removed?

Reply:
Removed in d210b2c8a786a046301efaa93f37a2bbc0e3229a

Reply:
Thanks @joshsadam.

I just noticed there's two methods which are unused in this class too `extractPipelineParameters` and `getCartSummaryMap`. Could you check to make sure these aren't required and remove them if so?

Reply:
Good catch, removed in aa2c631d9e276eb6e2b17542e8ce42304eac1a82

Reply:
Looks great. Thanks @joshsadam